tests/unit/test__opentelemetry_tracing.py

```diff
@@ -0,0 +1,129 @@
+import importlib
+import mock
+import unittest
+import sys
+
+try:
+    from opentelemetry import trace as trace_api
+    from opentelemetry.trace.status import StatusCanonicalCode
+except ImportError:
+    pass
+
+from google.api_core.exceptions import GoogleAPICallError
+from google.cloud.spanner_v1 import _opentelemetry_tracing
+
+from tests._helpers import OpenTelemetryBase, HAS_OPENTELEMETRY_INSTALLED
+
+
+def _make_rpc_error(error_cls, trailing_metadata=None):
+    import grpc
+
+    grpc_error = mock.create_autospec(grpc.Call, instance=True)
+    grpc_error.trailing_metadata.return_value = trailing_metadata
+    return error_cls("error", errors=(grpc_error,))
+
+
+def _make_session():
+    from google.cloud.spanner_v1.session import Session
+
+    return mock.Mock(autospec=Session, instance=True)
+
+
+# Skip all of these tests if we don't have OpenTelemetry
+if HAS_OPENTELEMETRY_INSTALLED:
```

Comment:
I think we should get rid of this. Instead, we should make sure that OpenTelemetry is installed. If not, the test should fail.

Reply:
see above comment on py2.7, same thing applies here